### ISSUE TITLE
Create page "Conditions of Use" under menu About

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/Global-signbank#461]

**Comment by Woseseltops** [comment]
Done by clicking around in the admin... forgot this was possible without any code.

![](https://img.buzzfeed.com/buzzfeed-static/static/2016-01/14/7/enhanced/webdr08/anigif_original-grid-image-3265-1452774333-1.gif?crop=425:281;75,0})

**Comment by ocrasborn** [comment]
Yes, right, I could do that myself, I forgot. But... I don't see the one you created. I currently see these:


/about/copyright/ | Copyright | /about/ -- About | 0
  | Guidelines for new signs | /about/ -- About | 2
  | Phonology: Handshape changes | /about/ -- About | 4
  | Phonology: Handshapes | /about/ -- About | 3
  | Phonology: Locations | /about/ -- About | 5
  | Manual | /about/ -- About | 1

**Comment by Woseseltops** [comment]
> But... I don't see the one you created

Ah sorry I didn't read that it should also be done for the Global SB. Done that too.

Question by Julie:

> I want the tab "conditions of use" under the "about" tab to be available to all users of the site including the public access view

I deselected any required groups for the page, and now it's available even when logged out.

Happy @ocrasborn ?
